generators/app/templates/.storybook/webpack.config.js

```diff
@@ -1,3 +1,5 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
I believe we just added it because the linter was confused when we were importing storybook but its listed as a dev dependancy. We could instead of doing this globablly, just add the opt out comment to the stories/index.jsx file